features/support/helpers.rb

```diff
@@ -103,24 +102,17 @@ def run_jekyll(args)
 end
 
 #
-
-# rubocop:disable Metrics/AbcSize
 def run_in_shell(*args)
-  i, o, e, p = Open3.popen3(*args)
-  out = o.read.strip
-  err = e.read.strip
-
-  [i, o, e].each(&:close)
+  p, output = Jekyll::Utils::Exec.run(*args)
 
-  File.write(Paths.status_file, p.value.exitstatus)
+  File.write(Paths.status_file, p.exitstatus)
   File.open(Paths.output_file, "wb") do |f|
     f.puts "$ " << args.join(" ")
-    f.puts out
-    f.puts err
-    f.puts "EXIT STATUS: #{p.value.exitstatus}"
+    f.puts output
+    f.puts "EXIT STATUS: #{p.exitstatus}"
   end
 
-  p.value
+  p
 end
 # rubocop:enable Metrics/AbcSize
```

Comment:
Since we removed `rubocop:disable`, we could probably remove `rubocop:enable` as well